content/master/composition/compositions.md

```diff
@@ -767,6 +767,185 @@ Crossplane doesn't validate function input. It's a good idea for a function to
 validate its own input.
 {{</hint>}}
 
+### Required resources
+
+{{<hint "note">}}
+Crossplane v1 called this feature "extra resources." The v2 API
+uses the name "required resources" and adds support for bootstrap requirements.
+Functions now support the new API - function-python
```

Comment:
is function-python meant as an example? it's sort of reading like it's the one authoritative implementation 

perhaps a "for example" here could help?

Reply:
Whoops, yeah I noticed this too and thought I fixed it. Will fix.

Reply:
I just deleted it.

It was originally trying to say that only function-python really supports this right now (AFAIK). In retrospect I don't wanna say that though - it'll quickly become outdated.